actors.md

```diff
@@ -708,3 +708,233 @@ func UpdatePeerID(pid PeerID) {
 ## Payment Channel Broker Actor
 
 TODO
+
+## Multisig Account Actor
+
+A basic multisig account actor. Allows sending of messages like a normal account actor, but with the requirement of M of N parties agreeing to the operation. Any party can veto any operation (which is a potentially debateable design decision). Completed and/or cancelled operations stick around in the actors state until explicitly cleared out.
+
```

Comment:
Meta note, this kind of multi-sig is ok, but since we use BLS signature already anyway, why don't we use threshold signatures instead? This would reduce multisig on-chain footprint. (I might be missing context on threshold sig vs on-chain multi-sig).